src/datastore/postgres-store.ts

```diff
@@ -5860,9 +5860,10 @@ export class PgDataStore
       tx_id: Buffer;
     }>(
       `
-      SELECT locked_amount, unlock_height, block_height, tx_id, locked_address
-      FROM stx_lock_events
-      WHERE canonical = true AND unlock_height <= $1 AND unlock_height > $2
+      SELECT locked_amount, unlock_height, sle.block_height, locked_address, txs.tx_id
+      FROM stx_lock_events sle inner JOIN txs ON (sle.unlock_height = txs.block_height)
```

Comment:
It looks like `sle.unlock_height` is referring to the burnblock (btc) height whereas `txs.block_height` is referring to Stacks block height, so this JOIN won't work correctly

Reply:
oh, in that case probably need another join.